### PR TITLE
[next-devel] manifest: move to Fedora 34

### PR DESCRIPTION
**Pull request by jlebon**
Fedora 34 Beta is GO for March 23:
https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/H6CDDLGK7KWEFGYB2PWOVSUXTYWIH2MA/

As per https://github.com/coreos/fedora-coreos-tracker/issues/704, we
want to move `next` over to it in the next release. So let's move over
`next-devel` now.

### CODE REVIEW

**Comment by jlebon** [comment]
Leaving as draft for now. Pretty sure there's still some SELinux violations lingering.

**Comment by jlebon** [comment]
CI is green at least, :tada: 

That said, for some reason `cosa run` can't SSH into the VM. Debugging.

**Comment by jlebon** [comment]
> That said, for some reason `cosa run` can't SSH into the VM. Debugging.

Fix in https://github.com/coreos/coreos-assembler/pull/2094.

**Comment by jlebon** [comment]
OK, this is ready now! There are a few other SELinux failures, which I've reported at:
- https://bugzilla.redhat.com/show_bug.cgi?id=1942205
- https://github.com/fedora-selinux/selinux-policy/pull/651

But none of them majorly affect host stability.

File: manifest-lock.x86_64.json [hunk 1]
```diff
@@ -1,1226 +1,1211 @@
...
-      "evra": "4.1-6.fc33.x86_64"
+      "evra": "4.1-7.fc34.x86_64"
     },
     "microcode_ctl": {
-      "evra": "2:2.1-43.1.fc33.x86_64"
+      "evra": "2:2.1-45.fc34.x86_64"
     },
     "moby-engine": {
-      "evra": "19.03.13-1.ce.git4484c46.fc33.x86_64"
+      "evra": "20.10.5-1.fc34.x86_64"
```
Comment (jlebon) [inline manifest-lock.x86_64.json:865]: Worth highlighting here that as expected this includes `moby-engine` v20.10 which unblocks the move to cgroupsv2 (https://github.com/coreos/fedora-coreos-tracker/issues/292).

**Review by cgwalters — APPROVED** [review]
Thanks for taking this on!

**Review by miabbott — APPROVED** [review]
Looks sane to me!  Here's to cgroups v2! :clinking_glasses:

**Review by sohankunkerkar — COMMENTED** [review]
LGTM, but needs a rebase on top of the latest HEAD.

**Review by travier — APPROVED** [review]

**Pull request merged**